tests/customer-resource-embargo-test.js

```diff
@@ -82,10 +82,10 @@ describeApplication('CustomerResourceEmbargo', () => {
 
       describe('entering valid custom embargo value and selecting unit', () => {
         beforeEach(() => {
-          return convergeOn(() => {
-            expect(CustomerResourceShowPage.$customEmbargoTextField).to.exist;
-            expect(CustomerResourceShowPage.$customEmbargoSelect).to.exist;
-          });
+          return new Convergence()
+            .once(() => expect(CustomerResourceShowPage.$customEmbargoTextField).to.exist)
+            .once(() => expect(CustomerResourceShowPage.$customEmbargoSelect).to.exist)
```

Comment:
I won't point out every `.run()` but the `.once().once()` may be harder to find & replace